moped-editor/src/views/projects/projectsListView/useProjectListViewQuery/useSearch.js

```diff
@@ -28,10 +31,11 @@ export const getSearchValue = (column, value, queryConfig) => {
 };
 
 export const useSearch = ({ queryConfig }) => {
-  // create URLSearchParams from url
-  const navigationSearchTerm = useLocation()?.state?.searchTerm;
+  /* Get simple search from search params if it exists */
+  let [searchParams] = useSearchParams();
+  const simpleSearchValue = searchParams.get(simpleSearchParamName);
 
-  const [searchTerm, setSearchTerm] = useState(navigationSearchTerm ?? "");
```

Comment:
We used to pass the search term from the global search through [React Router link state](https://reactrouter.com/en/main/components/link#state). This PR adds a search parameter to the url that we can now consume here instead.